server/util/invocationutil/invocationutil.go

```diff
@@ -0,0 +1,34 @@
+package invocationutil
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	patternCharLimit = 50
+	listSeparator    = ", "
```

Comment:
Please comment these so that someone else could figure out how to set them.

Reply:
Done